my-app/src/utils/Regex.tsx

```diff
@@ -5,3 +5,35 @@ export const validateEmail = new RegExp(
 export const validatePassword = new RegExp(
   "^(?=.*?[A-Za-z])(?=.*?[0-9]).{7,}$",
 );
+
+export const validateBirthDate = (birthDate: string) => {
+  const regex = /^\d{2}\/\d{2}\/\d{4}$/;
+  if (!regex.test(birthDate)) {
+    return false;
+  }
+
+  const [day, month, year] = birthDate.split("/").map(Number);
+
+  const today = new Date();
+  const dia = today.getDate() + 1;
+  console.log(dia);
+  console.log(day);
```

Comment:
Acho que os logs não precisa commitar.

Reply:
to sempre esquecendo de apagar, uso pra uns testes... vou me atentar mais para isso